### PR TITLE
Add docs for static python3.10 build

### PR DESCRIPTION
**Pull request by OleksiiPiskun**

### CODE REVIEW

File: python3.10/README.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+Python 3.10.0 static compilation
```
Comment (HellCreator) [inline python3.10/README.md]: Use markup language to make article titles. you may use this tool https://dillinger.io/

File: python3.10/README.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+Ubuntu 18.04 VM with 4Gb RAM 2 VCPU
+
+configuration stored in vm_dumpxml.xml file
+
+uname -a output:
+
+Linux ubuntu18-buildserver 4.15.0-88-generic #88-Ubuntu SMP Tue Feb 11 20:11:34 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
+
+
```
Comment (HellCreator) [inline python3.10/README.md]: too much new lines

File: python3.10/README.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+
+Build preparation process:
+
+Install needed packages from file vm_installed_packages.txt
+
+while read -r line; do sudo apt-get -y install "$line"; done < vm_installed_packages.txt
```
Comment (HellCreator) [inline python3.10/README.md]: you do not need sudo if you login as root

File: python3.10/README.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+
+
+
```
Comment (HellCreator) [inline python3.10/README.md]: prepare build dir and clone repo there with all archives

File: python3.10/README.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+make install
+
+	RESULT: /usr/local/ssl1/lib/libssl.a
+
+
+
+TIP: To choose required ssl version:
+
+	Download python 3.10
```
Comment (HellCreator) [inline python3.10/README.md:55]: add python installation command apt ....

File: python3.10/README.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+Phase IV Build Python static binnary
+
+
+tar -zxvf Python-3.10.0.tgz
+
+mkdir build && cd build
+
+../configure --with-static-libpython --enable-optimizations --with-ensurepip=no --prefix=/usr/pythoncontroller --disable-shared --disable-test-modules LDFLAGS="-static" CFLAGS="-static -idirafter $MUSL_PATH/include/ -s -Os -g0" CPPFLAGS="-static -idirafter $MUSL_PATH/include/ -s -Os -g0"
+
+	Provide actual Setup.local file after configure and before make. It could be copied from previous versions and edited.
```
Comment (HellCreator) [inline python3.10/README.md]: should we include edited and working Setup.local ?

File: python3.10/README.md [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+
+
+
+Phase V Prepare python to OnApp
+
+	Copy web related site-packages into lib folder.
+
+	Remove unused libraries to optimize size:
+
+		 config-3.10-x86_64-linux-gnu, pydoc_data, lib2to3, tkinter, turtledemo, zoneinfo, ensurepip
```
Comment (HellCreator) [inline python3.10/README.md]: maybe it is better to add actual rm commands and also not forget about dynload folder creation and .git file inside of it

File: python3.10/README.md [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+
+	Now python should work correctly with this libs from /usr/pythoncontroller directory.
+
+
+
+
+
+Phase V Prepare python to OnApp
+
+	Copy web related site-packages into lib folder.
```
Comment (HellCreator) [inline python3.10/README.md]: we can add link to actua web lib we using

**Review by HellCreator — CHANGES_REQUESTED** [review]

File: python3.10/README.md (hunk 5 above)
Comment (OleksiiPiskun) [reply python3.10/README.md:55]: Default apt repo doesn't contain last python. 
python3.10 uses openssl-1.1.1l.
This tip is for future builds with next versions of python when openssl version will be unknown again.

File: python3.10/README.md (hunk 6 above)
Comment (OleksiiPiskun) [reply python3.10/README.md]: It is already included with packages that required for OnApp.

**Review by HellCreator — APPROVED** [review]

**Pull request merged**